sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -93,6 +93,8 @@ Scheduler::GraphBuilder::GraphBuilder() {
 }
 
 static bool markNodeAsVisited(Command *Cmd, std::vector<Command *> &Visited) {
+  if (!Cmd)
+    return false;
```

Comment:
So far no crashes were observed. So, if I understand correctly, Cmd might be equal to nullptr if something went wrong. What do you think about replacing this silent return with some verbose message (assert or exception)?